types/index.d.ts

```diff
@@ -226,7 +226,7 @@ export interface Provider {
 type CommonArguments = {
   dryRun: boolean // whether the CLI run as dryRun mode
   normalize?: string // normalization ways for locale messages or resource
-  providerArgs?: string // parameters to give to the provider
+  providerArgs?: object // parameters to give to the provider
```

Comment:
I think we should be defined empty `interface` type (e.g. `interface DefineCusomeArgs {}`.  because we can annotated with enhanced type definition (e.g. interface DefineCusomeArgs { x: string; y: string; }`)

Reply:
Or, We should use Generics.
For example:
```ts
type CommonArguments<T extends Record<string, any> = {}> = {
  providerArgs?: T
}
```

Reply:
Sure!
Wait a minute, please!